framework/ui/view/musicalsymboliconcodes.h

```diff
@@ -0,0 +1,70 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2020 MuseScore BVBA and others
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2.
+//
+//  This program is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+//  GNU General Public License for more details.
+//
+//  You should have received a copy of the GNU General Public License
+//  along with this program; if not, write to the Free Software
+//  Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+//=============================================================================
+
+#ifndef MU_FRAMEWORK_MUSICALSYMBOLICONCODE_H
+#define MU_FRAMEWORK_MUSICALSYMBOLICONCODE_H
+
+#include <QObject>
+
+namespace mu {
+namespace framework {
+/**
+ * @brief The MusicalSymbolIconCode class simplifies access to the icons from the musical icon font
+ *
+ * @details Each enum value is a UTF-16-like address of the icon in the icon font.
+ */
+
+class MusicalSymbolIconCode
+{
+    Q_GADGET
+
+public:
+    enum class Code : char16_t {
+        ZERO = 0xE080,
+        ONE = 0xE081,
+        TWO = 0xE082,
+        THREE = 0xE083,
+        FOUR = 0xE084,
+        FIVE = 0xE085,
+        SIX = 0xE086,
+        SEVEN = 0xE087,
+        EIGHT = 0xE088,
+        NINE = 0xE089,
+        TIMESIG_COMMON = 0xE08A,
+        TIMESIG_CUT = 0xE08B,
+
+        SEMIBREVE = 0xE1D2,
+        MINIM = 0xE1D3,
+        CROTCHET = 0xE1D5,
+        QUAVER = 0xE1D7,
+        SEMIQUAVER = 0xE1D9,
+        DEMISEMIQUAVER = 0xE1DB,
+        DOT = 0xE1E7,
+
+        AUTO,
```

Comment:
I'm not sure that we need it here
If you need auto, none and custom you'd prefer to use IconCode instead